csrf/api/v1/views.py

```diff
@@ -27,8 +28,11 @@ class CsrfTokenView(APIView):
     """
 
     authentication_classes = (JwtAuthentication,)
-    permission_classes = (IsAuthenticated,)
 
+    # Use requires_csrf_token on views that need a correct csrf_token, but without the
+    # CSRF protection that csrf_protect enforces.
```

Comment:
Can you explain this statement?  I'm having trouble parsing this sentence.  Perhaps it's a tongue-twister with the c's and the s's, etc.

Reply:
Although comments made me think this was needed, it seems to work without this, so I removed this.